doc/README.md

```diff
@@ -1,5 +1,24 @@
 # Composite Setup Guide
 
+## Tools nessesary for Composite
+```
+sudo apt-get -y install bc  
+sudo apt-get -y install gcc-multilib  
+sudo apt-get -y install binutils-dev  
+sudo apt-get -y install qemu-kvm  
+sudo apt-get -y install g++  
+sudo apt-get -y install make  
+sudo apt-get -y install cmake  
```

Comment:
Also, `sudo apt-get install build-essential` should take care of `gcc`, `g++`, `make`, which is what I usually install. Package info: https://packages.ubuntu.com/trusty/devel/build-essential.